tests/Microsoft.Bot.Builder.Transcripts.Tests/Microsoft.Bot.Builder.Transcripts.Tests.csproj

```diff
@@ -8,8 +8,6 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.1.1" />
-    <PackageReference Include="MSTest.TestAdapter" Version="1.4.0" />
-    <PackageReference Include="MSTest.TestFramework" Version="1.4.0" />

```

Comment:
> <PackageReference Include="MSTest.TestFramework" Version="1.4.0" /> [](start = 3, length = 68)

We should add the XUnit runner so tests can run locally.

Reply:
added